storage/replica_command.go

```diff
@@ -1426,13 +1426,18 @@ func (r *Replica) splitTrigger(batch engine.Engine, ms *engine.MVCCStats, split
 		return util.Errorf("unable to write MVCC stats: %s", err)
 	}
 
-	// Copy the last verification timestamp.
-	verifyTS, err := r.GetLastVerificationTimestamp()
+	// Copy the last replica GC and verification timestamps.
+	replicaGCTS, err := r.getLastReplicaGCTimestamp()
+	if err != nil {
+		return util.Errorf("unable to fetch last replica GC timestamp: %s", err)
+	}
+	if err := engine.MVCCPutProto(batch, nil /* &deltaMs */, keys.RangeLastReplicaGCTimestampKey(split.NewDesc.RangeID), roachpb.ZeroTimestamp, nil, &replicaGCTS); err != nil {
+		return util.Errorf("unable to copy last replica GC timestamp: %s", err)
+	}
+	verifyTS, err := r.getLastVerificationTimestamp()
 	if err != nil {
 		return util.Errorf("unable to fetch last verification timestamp: %s", err)
 	}
-	// TODO(nvanbenschoten) update stats again here when #4710 is resolved
-	// and the LastVerificationTimestampKey is replicated again.
 	if err := engine.MVCCPutProto(batch, nil /* &deltaMs */, keys.RangeLastVerificationTimestampKey(split.NewDesc.RangeID), roachpb.ZeroTimestamp, nil, &verifyTS); err != nil {
```

Comment:
I think the `/* &deltaMS */` should be removed (it was a reminder for @nvanbenschoten as to what it should look like after addressing the TODO). Same in line 1434.